packages/perseus/src/util/interactive.ts

```diff
@@ -969,6 +969,7 @@ _.extend(GraphUtils.Graphie.prototype, {
                     radii,
                     options,
                 );
+                movablePoint.mouseTarget.wrapper.setAttribute("tabindex", "0")
```

Comment:
We have some designs, don't we? Even if we get the browser focus defaults, this won't give us consistency across browsers. Let's look at our designs and at least give that a shot